2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -214,13 +215,149 @@ Viene inserito ";" alla fine delle voci dell'elenco tranne l'ultima che termina
+\ \
+===== Verifica
+Al termine di ogni sprint, il gruppo procederà con il meeting di retrospettiva, durante il quale verranno analizzati i risultati ottenuti e le attività svolte, basandosi sui prodotti della fase di esecuzione: \
++ grafici di burndown:
+  - Permetto di avere una visione rapida di quanto del lavoro pianificato è stato portato a termine: si tratta di una metrica puramente quantitativa, che tiene conto di del numero di ticket chiusi e del numero di ticket aperti;
++ board di avanzamento:
+  - "Keep doing": attività che hanno dato buoni risultati e che quindi vanno mantenute: questo permette al gruppo di individuare il modus operandi più efficace ed efficiente per svolgere le attività;
+  - "Things to change": attività che hanno dato risultati non soddisfacenti e che quindi vanno modificate: si tratta dell'aspetto più delicato da considerare, in quanto permette di individuare le criticità e le inefficienze del processo di sviluppo, e di conseguenza di apportare le modifiche necessarie per migliorare il processo stesso.
+
+Questa analisi individua i miglioramenti da apportare al processo di sviluppo, e di conseguenza di stabilire le contromisure necessarie per migliorare il processo stesso.
+  \ \
+===== Attuazione
+L'attuazione è l'ultima fase del processo di gestione del ciclo di vita, e consiste nell'attuare le contromisure stabilite durante la fase di verifica. \
+L'obiettivo è sopperire alle mancanze e alle inefficienze del processo di sviluppo, in modo da migliorare la qualità del prodotto e la produttività del gruppo. \
+Diventa compito del responsabile stabilire le attività necessarie per attuare le contromisure, e di conseguenza di pianificare le attività da svolgere nello sprint successivo. \
+Il risultato di questo processo è visibile all'interno del Piano di Progetto.
+\
+\
+==== Ruoli 
+===== Responsabile
+Il responsabile è la figura chiave che guida il progetto, rappresentandolo nei confronti degli stakeholders. Le sue responsabilità sono molteplici:
+
++ coordinamento: ha il compito di supervisionare i membri del gruppo, assicurandosi che le attività siano svolte in modo efficiente e tempestivo;
+
++ pianificazione: il responsabile è incaricato di delineare il percorso del progetto, stabilendo le attività da svolgere, le relative scadenze e priorità, sancendo l'inizio e la fine di ogni sprint;
+
++ monitoraggio e gestione dei costi: il responsabile tiene sotto controllo l'andamento del progetto, stima i costi e gestisce l'analisi dei rischi, garantendo che il progetto rimanga entro il budget previsto;
+
++ norme di progetto: il responsabile è responsabile della stesura e dell'aggiornamento delle norme di progetto, che devono essere rispettate da tutti i membri del gruppo;
+
++ relazioni esterne: gestisce tutte le interazioni con la proponente e i committenti assicurando una comunicazione fluida e efficace;
```

Comment:
suppongo tu intenda l'azienda proponente per questo hai usato il femminile, però noi nei documenti ci siamo sempre riferiti al proponente in versione maschile.